CHANGELOG.md

```diff
@@ -64,6 +64,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (x/wasm) [\#649](https://github.com/line/lbm-sdk/pull/649) fix: wasm module's FIXME in the snapshotter.go file
 * (x/ibc) [\#651](https://github.com/line/lbm-sdk/pull/651) feat: update x/ibc to support github.com/cosmos/ibc-go@v3.0.0
 * (config) [\#665](https://github.com/line/lbm-sdk/pull/665) remove bech32-cache-size
+* (x/wasm) [\#948](https://github.com/line/lbm-sdk/pull/948) feat: Implement CallCallablePoint Function to wasm Keeper
```

Comment:
I think CHANGELOG to the dynamic_link branch should be added to `Unreleased`.

What do you think? @loloicci

Reply:
Thanks. Certainly as you pointed out, I think it would be better to include it in Unreleased as well. What do you think? @loloicci

Reply:
Since CHANGELOG.md is not organized, an issue was created to address this issue separately. We will leave it as is in this PR.
https://github.com/line/lbm-sdk/issues/950